glfs.c

```diff
@@ -641,5 +682,12 @@ struct tcmur_handler glfs_handler = {
 /* Entry point must be named "handler_init". */
 int handler_init(void)
 {
+	int ret;
+
+	ret = pthread_mutex_init(&glfs_lock, NULL);
+	if (ret != 0) {
+		return -1;
+	}
+
 	return tcmur_register_handler(&glfs_handler);
```

Comment:
Hey, sorry just noticed this. If tcmur_register_handler fails, we leak whatever pthread_mutex_init allocated. Need a pthread_mutex_destroy in the error path.